src/app/Instance/InstancesListPage/InstancesListPage.tsx

```diff
@@ -8,23 +8,27 @@ import {
   Text,
   TextContent,
 } from "@patternfly/react-core";
-import { IRow, IRowData } from "@patternfly/react-table";
+import { IRowData } from "@patternfly/react-table";
 import { Link } from "react-router-dom";
 import CreateInstance from "@app/Instance/CreateInstance/CreateInstance";
+import { Instance } from "../../../types/Instance";
 
 const InstancesListPage = () => {
-  const columnNames = [
+  const columnNames: TableColumn[] = [
     {
       accessor: "name",
       label: "Name",
-      formatter: (value: IRowData, row?: IRow) => (
-        <Link
-          data-testid="tableInstances-linkInstance"
-          to={`instance/${row?.id}`}
-        >
-          {value}
-        </Link>
-      ),
+      formatter: (value: IRowData, row?) => {
```

Comment:
It seems weird that we're mandating "typing" e.g. [here](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/15/files#diff-0e766af1b39216dbeff5e7d8c1965bf8ad708f1c98f9ff42edb1ee55197977c2R17) but on this line we're actually removing the typing and casting on the next line.

Reply:
@manstis I am the author of this change.
The type of `row` is inferred to be `IRow` (it is not changed).
On line 22 I casted to `Instance` to give it more context related to this page (and in order to access to `id` property).
In my opinion it was not needed to have the type explicit in the function argument, since it is casted in line 22.
But feel free to restore it, if you prefer.

Reply:
@vpellegrino OK. I thought we were mandating use of type information on this PR with linting rules.

If this is not the case then please ignore my comment.

Reply:
I added `row?: IRow` in a separate commit

Reply:
@manstis fair point. I was trying to give more context. I am always willing to learn new things and make the code more easy to read.